R/annotate_signature.R

```diff
@@ -1,61 +1,153 @@
 #' Gene set fetch
 #'
-#' Fetches GO or KEGG gene sets from the Tribe webserver
-#' (url{tribe.greenelab.com}).
+#' Fetches gene sets from the Tribe webserver
+#' (url{https://tribe.greenelab.com}). It can retrieve GO, KEGG, or user-created
+#' public gene sets for Pseudomonas aeruginosa.
 #'
-#' @param type character, type of the gene sets, either "GO" or "KEGG"
+#' @param type character, type of the gene sets, can be "GO", "KEGG", or "User"
 #' (default: "KEGG").
+#' @param username character, creator's TRIBE username (default: NULL)
+#' @param access_date character, in the format of month-day-year (12-31-16).
```

Comment:
Should we say here that the year must only contain the last two digits (or else it will complain about an incorrect format?)